libs/designsystem/dropdown/src/dropdown.component.ts

```diff
@@ -290,6 +292,11 @@ export class DropdownComponent implements AfterViewInit, OnDestroy, ControlValue
       this.setPopoverCardStyle('--kirby-card-width', `${width}px`);
```

Comment:
Do we still need to call `this.setPopoverCardStyle('--kirby-card-width')` here, or can it always be handled through the resize observer callback?

Reply:
Probably not, but specs doesn't explicitly state that the resize observer callback is always called initially 🤷🏻 
Added a comment about better safe than sorry...